roctracer/PKGBUILD

```diff
@@ -11,18 +11,25 @@ makedepends=('cmake' 'git' 'python' 'python-argparse' 'python-cppheaderparser' '
 options=(!staticlibs strip)
 _git='https://github.com/ROCm-Developer-Tools/roctracer'
 source=("roctracer-rocm-$pkgver.tar.gz::$_git/archive/rocm-$pkgver.tar.gz"
+        'deprecated_string_split.patch'
```

Comment:
Is it not possible to use the code from upstream?

Reply:
I don't know. The upstream version of the script was completely rewritten after the ROCm 3.8.0 release (Based on other commits, I suppose that this is in preparation for ROCm 4.0.0).